### PR TITLE
Remove type annotation from majority.py example

### PR DESCRIPTION
**Pull request by luxusko**
Provided type cx.AbstractDataset.Stream no longer exists in cxflow.

### CODE REVIEW

File: docs/tutorial.rst [hunk 1]
```diff
@@ -94,12 +94,12 @@ To generate the *majority* data and provide the data streams we will implement a
             self._train_x, self._train_y = x[:int(.8 * n_examples)], y[:int(.8 * n_examples)]
             self._test_x, self._test_y = x[int(.8 * n_examples):], y[int(.8 * n_examples):]
 
-        def train_stream(self) -> cx.AbstractDataset.Stream:
+        def train_stream(self):
```
Comment (petrbel) [inline docs/tutorial.rst]: Why can't we use `cx.Stream` instead?

File: docs/tutorial.rst [hunk 2]
```diff
@@ -94,12 +94,12 @@ To generate the *majority* data and provide the data streams we will implement a
...
             for i in range(0, len(self._train_x), self.batch_size):
                 yield {'x': self._train_x[i: i + self.batch_size],
                        'y': self._train_y[i: i + self.batch_size]}
 
-        def test_stream(self) -> cx.AbstractDataset.Stream:
+        def test_stream(self):
```
Comment (petrbel) [inline docs/tutorial.rst]: ditto

File: docs/tutorial.rst (hunk 1 above)
Comment (blazekadam) [reply docs/tutorial.rst]: We can.

Comment (blazekadam) [reply docs/tutorial.rst]: And we should :)

**Pull request merged**